installer/setup.go

```diff
@@ -270,7 +269,7 @@ func main() {
 		fmt.Println("not known platform")
 		os.Exit(3)
 	}
-	fileUrl := "https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/" + cliFileName
+	fileUrl := downloadUrl() + cliFileName
```

Comment:
Better to use `downloadUrl() + "/" + cliFileName` and remove / from end of line 293 to avoid error in case user missed to add / at the end of CLI_DRIVER_DOWNLOAD_URL.

Reply:
fixed according to your comment below